docs/concepts.md

```diff
@@ -0,0 +1,12 @@
+### Federation V2 Concepts
+
+| Concept | Description |
+|-----------|-------------|
+| Federate | Federating a set of Kubernetes clusters means, effectively creating a common interface to the pool of these federated clusters which can be used to deploy kubernetes applications across the same. |
```

Comment:
I would try avoiding using the concept name in the description. It also appears that you did not complete the last sentence.